salt/modules/win_system.py

```diff
@@ -977,7 +976,40 @@ def get_domain_workgroup():
         if computer.PartOfDomain:
             return {'Domain': computer.Domain}
         else:
-            return {'Workgroup': computer.Domain}
+            return {'Workgroup': computer.Workgroup}
+
+
+def set_domain_workgroup(workgroup):
+    '''
+    Set the domain or workgroup the computer belongs to.
+
+    .. versionadded:: 2019.2.0
+
+    Returns:
+        bool: ``True`` if successful, otherwise ``False``
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt 'minion-id' system.set_domain_workgroup LOCAL
+    '''
+    if six.PY2:
+        workgroup = _to_unicode(workgroup)
+
+    # Initialize COM
+    pythoncom.CoInitialize()
+
+    # Grab the first Win32_ComputerSystem object from wmi
+    conn = wmi.WMI()
+    comp = conn.Win32_ComputerSystem()[0]
+
+    # Grab the current workgroup/domain value
+    current = comp.Domain if comp.PartOfDomain else comp.Workgroup
```

Comment:
Ok. Cool. Removing it.

Btw, I had it there because I wasn't sure if it was better to return a dict showing the changes (previous name, and current name), or just a boolean which is similar to some of the others. Which one do you think/prefer?

Reply:
Yeah, the original code is older. We're just trying to keep from accruing more technical debt where possible.

Reply:
Ok. I'll change it to add more details